src/clients.rs

```diff
@@ -234,13 +229,15 @@ impl StateClient for State {
     /// Find notes for a view key, starting from the given block height.
     fn fetch_notes(
         &self,
-        vk: &ViewKey,
+        sk: &SecretSpendKey,
     ) -> Result<Vec<EnrichedNote>, Self::Error> {
+        let vk = sk.view_key();
         let mut state = self.inner.lock().unwrap();
 
         self.status("Getting cached block height...");
         let psk = vk.public_spend_key();
         let mut last_height = state.cache.last_height(psk)?;
+        println!("LAST HEIGHT: {}", last_height);
```

Comment:
Is this a leftover? If not, use `self.status` to display information

Reply:
Removed